changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [6.1.0] - 2019-11-25
+
```

Comment:
Can you note what was removed

Reply:
Refactored changelog msg to Updated/Removed